.travis.yml

```diff
@@ -8,9 +8,11 @@ go_import_path: github.com/Stackdriver/stackdriver-prometheus-sidecar
 # Skip dependency fetch. We store all dependencies under vendor/.
 install: true
 
-# make goveralls runs all tests as well.
 script:
-- make goveralls
+# Can't run goveralls from external pull requests.
+- 'if [[ "${TRAVIS_PULL_REQUEST_SLUG:-Stackdriver/stackdriver-prometheus-sidecar}" != "Stackdriver/stackdriver-prometheus-sidecar" ]]; then make test; fi'
+# make goveralls runs all tests as well.
```

Comment:
Maybe also comment that this runs goveralls when the job is a push build (i.e. when TRAVIS_PULL_REQUEST_SLUG is "").

Reply:
Done.